src/specifications/profile-resolution/profile-resolution-specml.xml

```diff
@@ -3,1772 +3,1370 @@
+          of object. Each subsection is a list of ID mappings to be applied for objects that are the
+          parent object type.</p>
+        <p>When encountering a given mapping instruction, processors:</p>
+        <ul>
+          <li>
+            <p>MUST locate the included object in the intermediate whose distinctive ID matches the
+              value of the "from" object and whose object type is the same as the name as the
+              parent. If none is found, the processor SHOULD provide a warning, but MAY continue
+              processing. In the case there are more than one included objects of the specified type
+              with the same distinctive ID, behavior is unspecified, as this means the source
+              catalog was malformed.</p>
+          </li>
+          <li>
+            <p>MUST change the distinctive ID of that object to be equal to the value of the "to"
+              object.</p>
+          </li>
+          <li>
+            <p>SHOULD from this point forward only consider (or even be aware of) the new value of
+              that object's distinctive ID.</p>
+          </li>
+        </ul>
+        <p>Since mapping is a self contained process inside each import, the rest of this
+          specification will continue to reference IDs with the assumption that mapping has already
+          been applied if it was present. Since mapping is most commonly used to avoid clashing IDs,
+          processors should take care to not handle duplicate IDs until after mapping is
+          complete.</p>
+      </section>
+      <section id="import-wrap-up">
+        <head>Wrapping up the Import Phase</head>
+        <p>At this point the processor should have an intermediate that contains: a set of explicitly
+          included controls and all of their child informational (non-control, non-group) objects; a
+          set of implicitly included controls and groups that were parents of the previous (zero to
+          many); and a set of implicitly included "loose params" <xref rid="import-param"/> (zero to
+          many).</p>
+
+        <p>The general structure of explicitly included controls should match that of those controls
+          in the imported catalogs (i.e. Nested controls remain nested, grouped controls remain
+          grouped).</p>
+
+      </section>
     </section>
     <section id="merge-phase">
-      <head>Merge phase</head>
+
+      <!-- TODO: Suggestion. Refactor merge into:
+        merge:
+          combine: 
+            {{keep or use-first}}
+              config
+          structure: {{flat, as-is, or custom}}
+            
+            /////
+              OR
+            /////
+            
+        merge:
+          combine: {{keep or use-first}}
+          structure: {{flat, as-is, or custom}}
+          
+      The object is way overcomplicated at the moment. 
+      
+      TODO: Add the above as a Github Issue
```

Comment:
@stephenbanghart I'd like to talk through this with you to see what we can do in the OSCAL 1.0.1 release.